base/array.jl

```diff
@@ -982,6 +991,10 @@ julia> prepend!([3],[1,2])
  2
  3
 ```
+
+Use [`unshift!`](@ref) to add individual items to `collection` which are not already
+themselves in another collection. The result is of the preceding example is equivalent to
```

Comment:
I find the phrase "which are not already themselves in another collection" confusing.  I think I know what you are getting at, but I would remove it.